segregation/aspatial/flycheck_aspatial_indexes.py

```diff
@@ -0,0 +1,2039 @@
+    if ((group_pop_var not in data.columns) or (total_pop_var not in data.columns)):
+        raise ValueError('group_pop_var and total_pop_var must be variables of data')
+
+    data = data.rename(columns={group_pop_var: 'group_pop_var',
+                                total_pop_var: 'total_pop_var'})
+
+    x = np.array(data.group_pop_var)
+    t = np.array(data.total_pop_var)
+
+    if any(t < x):
+        raise ValueError('Group of interest population must equal or lower than the total population of the units.')
+
+    T = t.sum()
+    P = x.sum() / T
+
+    # If a unit has zero population, the group of interest frequency is zero
+    pi = np.where(t == 0, 0, x / t)
+
+    D = (((t * abs(pi - P)))/ (2 * T * P * (1 - P))).sum()
+
+    if (str(type(data)) != '<class \'geopandas.geodataframe.GeoDataFrame\'>'):
+        core_data = data[['group_pop_var', 'total_pop_var']]
+
+    else:
+        core_data = data[['group_pop_var', 'total_pop_var', 'geometry']]
+
+    return D, core_data
+
+
+class Dissim:
+    """
+    Classic Dissimilarity Index
+
+    Parameters
+    ----------
+
+    data          : a pandas DataFrame
+
+    group_pop_var : string
+                    The name of variable in data that contains the population size of the group of interest
+
+    total_pop_var : string
+                    The name of variable in data that contains the total population of the unit
+
+    Attributes
+    ----------
+
+    statistic : float
+                Dissimilarity Index
+
+    core_data : a pandas DataFrame
+                A pandas DataFrame that contains the columns used to perform the estimate.
+
+    Examples
+    --------
+    In this example, we will calculate the degree of dissimilarity (D) for the Riverside County using the census tract data of 2010.
+    The group of interest is non-hispanic black people which is the variable nhblk10 in the dataset.
+
+    Firstly, we need to perform some import the modules and the respective function.
+
+    >>> import pandas as pd
+    >>> import geopandas as gpd
+    >>> import segregation
+    >>> from segregation.aspatial import Dissim
+
+    Secondly, we need to read the data:
+
+    >>> # This example uses all census data that the user must provide your own copy of the external database.
+    >>> # A step-by-step procedure for downloading the data can be found here: https://github.com/spatialucr/geosnap/blob/master/examples/01_getting_started.ipynb
+    >>> # After the user download the LTDB_Std_All_fullcount.zip and extract the files, the filepath might be something like presented below.
+    >>> filepath = '~/data/LTDB_Std_2010_fullcount.csv'
+    >>> census_2010 = pd.read_csv(filepath, encoding = "ISO-8859-1", sep = ",")
+
+    Then, we filter only for the desired county (in this case, Riverside County):
+
+    >>> df = census_2010.loc[census_2010.county == "Riverside County"][['pop10','tractid']]
+
+    The value is estimated below.
+
+    >>> dissim_index = Dissim(df, 'tractid', 'pop10')
```

Comment:
This could be handled in another PR, but just to flag there is a typo here and should not be "tractid" in the examples.